app/views/comments/new.html.haml

```diff
@@ -0,0 +1,5 @@
+%div{:id => "addCommentForm"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace